src/ray/protobuf/gcs.proto

```diff
@@ -56,7 +55,6 @@ enum TablePubsub {
   NODE_PUBSUB = 4;
   OBJECT_PUBSUB = 5;
   ACTOR_PUBSUB = 6;
-  HEARTBEAT_PUBSUB = 7;
   RESOURCE_USAGE_BATCH_PUBSUB = 8;
```

Comment:
Can you update the integer?

Reply:
e.g., `RESOURCE_USAGE_BATCH_PUBSUB =7`

Reply:
No problem. The reason why I didn't update them is that I saw the number below also has a gap.
I'll update all of them.